### PR TITLE
azure - datalake-analytics

### PR DESCRIPTION
**Pull request by dmytro-afanasiev**
Use case:
```yaml
policies:
  - name: asb_reslogs_datalakeanalytics
    description: |
      Data Lake Analytics with logging and retention policy disabled
    resource: azure.datalake-analytics
    filters:
      - or:
          - type: diagnostic-settings
            enabled: false
          - type: diagnostic-settings
            key: length(logs[?(category == 'Audit' || category == 'Requests') && enabled == `true` && retention_policy.enabled == `true` && retention_policy.days > `0`])
            value: 0
```

### CODE REVIEW

**Review by kapilt — APPROVED** [review]
thanks, lgtm

File: tools/c7n_azure/c7n_azure/resources/datalake_analytics.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    .. code-block:: yaml
+
+        policies:
+          - name: datalake-analytics
+            resource: azure.datalake-analytics
+
+    """
+    class resource_type(ArmResourceManager.resource_type):
+        doc_groups = ['Storage']
```
Comment (kapilt) [inline tools/c7n_azure/c7n_azure/resources/datalake_analytics.py:21]: doc group should be `Analytics`

Comment (dmytro-afanasiev) [reply tools/c7n_azure/c7n_azure/resources/datalake_analytics.py]: +

**Pull request merged**